### ISSUE TITLE
Stop Imagify 2.2+ from generating WebP images

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#308]
Imagify version 2.2 started automatically generating WebP images when it optimizes, with no interface for disabling it. 

We need to add a filter to stop this.

I started a github issue with them:

https://github.com/wp-media/imagify-plugin/issues/860

They provided this filter:

```
add_filter( 'imagify_nextgen_images_formats', function( $formats ) {
    if ( isset( $formats['webp'] ) ) {
        unset( $formats['webp'] );
    }

    return $formats;
} );
```

We may want to include logic so that it only applies the filter if "Serve images in next-gen format" is disabled (which it should be for clients, but you never know...)

**Comment by blogtutor** [comment]
Screenshot of the new settings, so you can see what I mean re: the "Display images in Next-Gen" setting. If that's enabled, we should still allow WebP images to be created (as much as I hate the bloat...)

![image](https://github.com/blogtutor/blog-tutor-support/assets/24479276/223ea476-ade1-4edd-a32a-78794482045f)

**Comment by blogtutor** [comment]
Just tested the latest branch on dev.blogtutor.com - works fine.

After doing so, I noticed this:
![image](https://github.com/blogtutor/blog-tutor-support/assets/24479276/d5112705-eff0-4fee-8f15-6dcc63069e8c)

I clicked the button and it gave me this popup:

![image](https://github.com/blogtutor/blog-tutor-support/assets/24479276/19962627-b5da-46b8-a931-4f76bf64df30)

The "Generate Missing Images" button went away. But then after reloading the settings page, it's back.

We should probably hide all this stuff to avoid confusion?   I'm okay with punting that to the next release for the sake of time, though.
